lib/executor/externally_controlled_test.go

```diff
@@ -62,50 +64,71 @@ func TestExternallyControlledRun(t *testing.T) {
 	defer cancel()
 
 	var (
-		wg            sync.WaitGroup
-		errCh         = make(chan error, 1)
-		doneCh        = make(chan struct{})
-		resultVUCount []int64
+		wg     sync.WaitGroup
+		errCh  = make(chan error, 1)
+		doneCh = make(chan struct{})
 	)
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
+		es.MarkStarted()
 		errCh <- executor.Run(ctx, nil)
+		es.MarkEnded()
 		close(doneCh)
 	}()
 
-	updateConfig := func(vus int) {
+	updateConfig := func(vus, maxVUs int, errMsg string) {
```

Comment:
seems to me that we could change them to `int64` without anything else being changed ?

Reply:
Indeed, thanks. Resolved in d84fe0ee.

I was trying to avoid having to explicitly use `int64()` everytime this is called, and forgot about the transparent `int` conversion.

Reply:
it's transparent `const` conversion ;)